pkg/operator/secretannotator/aws/reconciler.go

```diff
@@ -178,8 +220,55 @@ func (r *ReconcileCloudCredSecret) updateSecretAnnotations(secret *corev1.Secret
 		secretAnnotations = map[string]string{}
 	}
 
-	secretAnnotations[secretconstants.AnnotationKey] = value
+	secretAnnotations[constants.AnnotationKey] = value
 	secret.SetAnnotations(secretAnnotations)
 
 	return r.Update(context.Background(), secret)
 }
+
+// earlyAnnotationIfModeForced will read the CCO config, and go ahead and annotate the secret if
+// there is a forcedCredentialsMode set. It will return the fetched mode (empty string if no mode set),
+// whether the operator is in manual/disabled mode, and any errors encountered.
+func (r *ReconcileCloudCredSecret) earlyAnnotationIfModeForced(secretKey types.NamespacedName) (modeString string, disabled bool, err error) {
```

Comment:
Is there a reason why there needs to be both a `modeString` and `disabled` return value? Can we consolidate into just a `modeString` which is set to "manual" when the operator is disabled?

Reply:
now returning a "manual" mode when only the legacy configmap is set.